### ISSUE TITLE
New Dev UI show numbers in submenu for arc extension

### ISSUE THREAD

**Issue by bulldog98** [quarkusio/quarkus#32583]
### Description

The Arc extension benefits from seeing the number of beans as a badge in the sub-menu next to the page name (e.g. Beans).

This could also be useful for other extensions.

**Comment by mkouba** [comment]
@bulldog98 If I understand it correctly you would like to see the numbers shown in the extension card also in the submenu, right?

**Comment by bulldog98** [comment]
> @bulldog98 If I understand it correctly you would like to see the numbers shown in the extension card also in the submenu, right?

correct, that is what I intended to say

**Comment by phillip-kruger** [comment]
I agree . This will be a change to the Dev UI framework in general. So not ArC specific. I'll look at it a.s.a.p
